lib/classes/PluginManager.js

```diff
@@ -9,6 +9,21 @@ const getCacheFilePath = require('../utils/getCacheFilePath');
 const getServerlessConfigFile = require('../utils/getServerlessConfigFile');
 const crypto = require('crypto');
 
+/**
+ * @private
+ * Error type to terminate the currently running hook chain successfully without
+ * executing the rest of the current command's lifecycle chain.
+ */
+class TerminateHookChain extends Error {
+  constructor(commands) {
+    const commandChain = _.join(commands, ':');
+
+    super(commandChain);
+    this.message = commandChain;
```

Comment:
@pmuens Message could also be `Terminating ${commandChain}`, although this exception is only a control structure and never will lead to an exception on the outside.